WalletWasabi.Fluent/Views/Wallets/Advanced/WalletStatsView.axaml

```diff
@@ -17,7 +17,13 @@
                                  Content="{Binding CoinCount}" />
       </c:PreviewItem>
       <Separator />
-      <c:PreviewItem Label="Number of taproot coins (UTXOs)"
+      <c:PreviewItem Label="Number of segwit v0 (bech32) coins"
+                    TextValue="{Binding Bech32CoinCount}">
+        <c:PrivacyContentControl Classes="monoSpaced"
+                                 Content="{Binding Bech32CoinCount}" />
+      </c:PreviewItem>
+      <Separator />
+      <c:PreviewItem Label="Number of taproot coins"
```

Comment:
```suggestion
      <c:PreviewItem Label="Number of Taproot (bech32m) coins"
```

Reply:
I will wait until there is an agreement on this https://github.com/zkSNACKs/WalletWasabi/pull/10221#pullrequestreview-1328078085.